src/olympia/devhub/views.py

```diff
@@ -1225,7 +1226,10 @@ def auto_sign_version(version, use_autograph=False, **kwargs):
 
 @dev_required
 def version_list(request, addon_id, addon):
-    qs = addon.versions.order_by('-created').transform(Version.transformer)
+    versions = addon.versions
+    if not waffle.switch_is_active('beta-versions'):
+        versions = versions.exclude(files__status=amo.STATUS_BETA)
```

Comment:
@jvillalobos and I discussed briefly in person. I couldn't see a reason what the gain is for a developer to disable or delete their beta version.
Disabling only makes sense for beta versions to stop distributing updates, which does not apply since there will be no more submissions of beta versions.
Deleting has the same effect on developers than not showing them in the first place.

He mentioned that developers could want to download their beta versions for archival purposes. One idea was to include this in the notification email we send to developers upfront. 

But not final decision had been made.

@jvillalobos should beta versions be shown in the devhub on the versions page?

Reply:
Since the email to developers just went out, including that info is no longer an option.